Assets/MRTK/Tools/OptimizeWindow/MixedRealityOptimizeWindow.cs

```diff
@@ -395,7 +399,11 @@ private void RenderSettingOptimizations()
                 EditorGUILayout.LabelField(ToolbarTitles[(int)ToolbarSection.Settings], MixedRealityStylesUtility.BoldLargeTitleStyle);
                 using (new EditorGUI.IndentLevelScope())
                 {
+#if UNITY_ANDROID
                     RenderSinglePassSection();
+#else
+                    RenderSinglePassInstancingSection();
+#endif
```

Comment:
I would also change this to 
RenderOptimalRenderingSection or DrawOptimalRenderingPathSection etc